app/tests.py

```diff
@@ -1,3 +1,73 @@
 from django.test import TestCase
+from django.urls import reverse
+from .models import *
 
 # Create your tests here.
+class AppTestCase(TestCase):
+    def setUp(self):
+        self.slider = Slider.objects.create()
+        self.brands = Brands.objects.create()
```

Comment:
Why you are adding all model test cases in a single TestCase. There must be separate test cases for each viewset and his operations.